ee/codegen/src/__test__/nodes/code-execution-node.test.ts

```diff
@@ -80,7 +82,14 @@ describe("CodeExecutionNode", () => {
           codeExecutionNodeCodeRepresentationOverride: override,
         });
 
-        const nodeData = codeExecutionNodeFactory();
+        const nodeData = codeExecutionNodeFactory({
+          // This code triggers some things with the way fern does escapes that we need to test is escaping correctly
+          // as it sometimes does not for inline mode which is used by vembda.
+          code:
+            "async function main(inputs: {\n  question: string\n}) {\n  inputs = {\n    " +
+            '"question": "{\\"text_explanation\\":\\"First, \\\\\\\\(\\\\\\\\frac{1}{40}\\\\\\\\).\\\\"\n  }\n  ' +
+            "const test = \"\\frac\".replace(/\\\\frac/g, '\\\\dfrac');\n  return {};\n} \n",
```

Comment:
[nit] could we parametrize the test case further so that we have a base case code (the Hello World) and this complex case?

[nit] Could we make the string _here_ multiline instead of the newlines to make the user provided code slightly more readable?